Automattic/woocommerce-payments
# Transaction Details → show dispute amount in shopper currency

Fixes #7373 

#### Changes proposed in this Pull Request

This PR reverts changes introduced in #7287 to now **show the dispute amount in the shopper currency**.

Integration tests from #7287 for the case of a multi-currency store where a shopper/transaction currency differs from the store's currency have been kept to be explicit about the expected outcome.

**Before**

![image](https://github.com/Automattic/woocommerce-payments/assets/3147296/01448d96-0976-49a0-b920-018cdbb4b2fd)


**After**

![image](https://github.com/Automattic/woocommerce-payments/assets/3147296/66cadd07-5a87-40d1-8d69-231f638faef6)

#### Testing instructions





1. Enable **Payments → Settings → Multi-currency**.
3. Visit **Payments → Overview** and note your store's deposit currency or currencies, visible as balance tabs.
4. Enable a non-store currency in **WooCommerce → Settings → Multi-currency**. Enable the currency switcher if using the Storefront theme (otherwise, show the front-end currency switcher via the block editor or another method). 
8. Create a **disputed transaction in a non-store currency** using the currency switcher and the card `4000000000000259` at checkout. The transaction details screen should show the dispute amount **in the shopper/transaction currency, matching the headline transaction currency**.


-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. **Changelog entry removed as this change is a revert of a pre-release change**

**Post merge**
